src/sentry/api/helpers/group_index/update.py

```diff
@@ -142,17 +142,22 @@ def get_current_release_version_of_group(group: Group, follows_semver: bool = Fa
     current_release_version = None
     if follows_semver:
         try:
-            # This sets current_release_version to the latest semver version associated with a group
-            associated_release_id = GroupRelease.objects.filter(
-                project_id=group.project.id, group_id=group.id
-            ).values_list("release_id")
-
-            current_release_version = (
-                get_semver_releases(group.project)
-                .filter(id__in=associated_release_id)
-                .values_list("version", flat=True)[:1]
-                .get()
-            )
+            if not features.has(
+                "organizations:releases-resolve-next-release-semver-fix", group.project.organization
+            ):
+                # This sets current_release_version to the latest semver version associated with a group
+                associated_release_id = GroupRelease.objects.filter(
+                    project_id=group.project.id, group_id=group.id
+                ).values_list("release_id")
+                current_release_version = (
+                    get_semver_releases(group.project)
+                    .filter(id__in=associated_release_id)
+                    .values_list("version", flat=True)[:1]
+                    .get()
+                )
+            else:
+                current_release_version = greatest_semver_release(group.project).version
```

Comment:
can this return `Release.DoesNotExist`? from my quick reading of the code it seems like it cannot, so maybe we should move the try to only be wrapped around the `if not has feature` block?